engine/graphics/src/dx12/graphics_dx12.cpp

```diff
@@ -2824,6 +2825,15 @@ namespace dmGraphics
         // return GetAssetFromContainer<DX12Texture>(g_DX12Context->m_AssetHandleContainer, texture)->m_UsageHintFlags;
     }
 
+    static uint8_t DX12GetTextureSliceCount(HTexture texture)
+    {
+        // TODO: mutex is missed?
+        // ScopedLock lock(g_DX12Context->m_AssetHandleContainerMutex);

```

Comment:
Yeah, not sure why the DX12 doesn't have it when the OpenGL backend does 🤔 
Perhaps create a separate ticket about it (after we ask Jhonny first)